pkg/broker/receiver_test.go

```diff
@@ -196,3 +200,11 @@ func makeRequest() *http.Request {
 	}
 	return req
 }
+
+func makeV01Request() *http.Request {
+	return makeRequest(`"0.1"`, `"1.0"`, "CE-EventType", "CE-Source")
+}
+
+func makeV02Request() *http.Request {
+	return makeRequest(`"0.2"`, `"2.0"`, "CE-Type", "CE-Source")
```

Comment:
Use `ce-type` and `ce-source` (lower cased) to match the [spec](https://github.com/cloudevents/spec/blob/v0.2/http-transport-binding.md#314-examples).

Reply:
Done. who ends up changing it to Ce-Type and Ce-Source?
Because we check for that in the maps...

Reply:
Go's HTTP header logic does that. If we actually used the http.Header struct, then it would correctly check regardless of case, but for some reason we extract the map from inside so need to be precise.

Reply:
sounds good! thanks!